custom/ilsgateway/zipline/helpers.py

```diff
@@ -0,0 +1,51 @@
+from pytz import timezone
+
+
+def format_date(datetime):
+    return timezone('Africa/Dar_es_Salaam').localize(datetime).strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
we've been trying to use commcarehq utils for all timezone conversions, so that this line would become:

    from corehq.util.timezones.conversions import ServerTime
    ServerTime(datetime).user_time(timezone('Africa/Dar_es_Salaam')).done().strftime('%Y-%m-%d %H:%M:%S')